docs/build-with-java.md

```diff
@@ -0,0 +1,261 @@
+## Build an IoT App with Java
+
+### Architecture
+
+In this section, we will walk through and explain the code for the different commands.
```

Comment:
In this section, we will walk you through the CarePet commands and explain the code behind them.